lib/PluginFindRecord/PluginFindRecordModal.js

```diff
@@ -190,13 +191,25 @@ class PluginFindRecordModal extends React.Component {
     }
 
     const mixedResultsFormatter = {
-      isChecked: record => (
-        <Checkbox
-          type="checkbox"
-          checked={Boolean(checkedMap[record.id])}
-          onChange={() => this.toggleRecord(record)}
-        />
-      ),
+      isChecked: record => {
+        const value = getRecordLabel?.(record) ?? record.id;
```

Comment:
I was thinking of record.rowIndex, since `id` is kind of machine-readable identifier